api/Dockerfile

```
@@ -0,0 +1,20 @@
+FROM elixir:1.14.4
+
+# create a directory and add the project files to it
+WORKDIR /root
+
```

suggestion: somewhere in here (at least before `mix phx.server`) you should set `MIX_ENV=prod`
```suggestion
ENV MIX_ENV=prod
```

---

so the mix env should always be prod inside of a docker container? in that scheme, we only ever use mix dev when launching from the command line?

from [this](https://github.com/mbta/lamp/pull/121#discussion_r1178201722) comment it sounds like there is a doc on this?